consensus/src/threshold_simplex/mod.rs

```diff
@@ -269,7 +269,7 @@ mod tests {
         // Create context
         let n = 5;
         let threshold = quorum(n);
-        let max_exceptions = 4;
+        let max_exceptions = 10;
```

Comment:
We should really be running these tests with more seeds (subtle changes to the order have uncovered that some constraints are pretty flaky).